src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -756,28 +758,18 @@ components:
         - printRequestSummaries
     PrintRequestSummary:
       title: PrintRequestSummary
-      type: object
       description: An object containing print request summary
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/UpdatedBy'
```

Comment:
I have introduced a new mandatory object type that can now be reused across both summary responses
ℹ️ The response structure hasn't changed, so this is not a breaking change